.github/workflows/build.yml

```diff
@@ -7,37 +7,42 @@ on:
   pull_request:
 
 jobs:
-  chrome:
-    runs-on: ubuntu-20.04
+  setup:
+    runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Use Node.js 16
-        uses: actions/setup-node@v1
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'npm'
+      - run: npm run install-ci
+      - uses: actions/setup-node@v3
         with:
-          node-version: 16.x
-      - uses: actions/cache@v2
+          node-version: 16
+          cache: 'npm'
+      - run: npm run install-ci
+
+  chrome:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
-          path: ~/.npm
-          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
-          restore-keys: |
-            ${{ runner.os }}-node-
+          node-version: 16
+          cache: 'npm'
+      - run: npm run install-ci
       - run: npm run release-chrome
       - name: Check bundle size
         run: du -h build/chrome/release/chrome-release-*.zip
   firefox:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Use Node.js 16
-        uses: actions/setup-node@v1
-        with:
-          node-version: 16.x
-      - uses: actions/cache@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
-          path: ~/.npm
-          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
-          restore-keys: |
-            ${{ runner.os }}-node-
+          node-version: 16
+          cache: 'npm'
+      - run: npm run install-ci
```

Comment:
I believe this is already run in the `release-firefox` script (via the Makefile). Same for `release-chrome` above.

Reply:
Ah nice; thanks! I spent an inordinate amount of time thinking there was something special about this workflow when splitting it out into dependent workflows.